pkg/commands/verifyish.go

```diff
@@ -0,0 +1,140 @@
+package commands
+
+import (
+	"flag"
+	"fmt"
+	"time"
+
+	"github.com/hashicorp/actions-go-build/pkg/build"
+	"github.com/hashicorp/composite-action-framework-go/pkg/json"
+)
+
+type verifyish struct {
+	buildish
+	staggerTime                 time.Duration
+	verificationBuildResultFile string
+
+	primary      build.ResultSource
+	verification build.ResultSource
+}
+
+func (v *verifyish) Flags(fs *flag.FlagSet) {
+	v.buildish.Flags(fs)
+	fs.DurationVar(&v.staggerTime, "staggertime", 5*time.Second, "min. time to wait after start of primary build")
```

Comment:
I read ``min.`` as minute at first. We should probably put the whole ``minimum`` word here.